index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+	<head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>Hello Microverse</title>
+        <link rel="stylesheet" href="styles.css">
+	</head>
+	<p class="text">"Hello Microverse!"</p>
+	</body>
```

Comment:
[OPTIONAL] Kindly make sure all your tags have their corresponding opening and closing tag. In this case, you missed the opening tag for the body. This is really important in order to avoid bugs in your projects.